Library/Homebrew/dev-cmd/livecheck.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+require "cli/parser"
+require "livecheck/livecheck"
+require "livecheck/strategy"
+
+module Homebrew
+  module_function
+
+  WATCHLIST_PATH = (
+    ENV["HOMEBREW_LIVECHECK_WATCHLIST"] ||
+    Pathname.new(Dir.home)/".brew_livecheck_watchlist"
+  ).freeze
+
+  def livecheck_args
+    Homebrew::CLI::Parser.new do
+      usage_banner <<~EOS
+        `livecheck` [<formulae>]
+
+        Check for newer versions of formulae from upstream.
+
+        If no formula argument is passed, the list of formulae to check is taken from `HOMEBREW_LIVECHECK_WATCHLIST`
+        or `~/.brew_livecheck_watchlist`.
+      EOS
+      switch "--full-name",
+             description: "Print formulae with fully-qualified names."
+      flag   "--tap=",
+             description: "Check the formulae within the given tap, specified as <user>`/`<repo>."
+      switch "--installed",
+             description: "Check formulae that are currently installed."
+      switch "--json",
+             description: "Output informations in JSON format."
+      switch "--all",
+             description: "Check all available formulae."
+      switch "--newer-only",
+             description: "Show the latest version only if it's newer than the formula."
+      conflicts "--debug", "--json"
+      conflicts "--tap=", "--all", "--installed"
+    end
+  end
+
+  def livecheck
+    args = livecheck_args.parse
+
+    if args.debug? && args.verbose?
+      puts ARGV.inspect
+      puts args
```

Comment:
Is it worth having both? Should this have a header?

Reply:
> Is it worth having both?

It looks like [you added `args` to this output](https://github.com/Homebrew/homebrew-livecheck/commit/3d63817fa3974d4ed5594141dfe6d0a2e5588c5e), so I would say it's up to you.

I imagine it would only be useful to have both if/when the `args` parsing isn't working properly. However, we can always add temporary output like this when we're debugging issues, so there may not be much value in the interim time.

If it were up to me, I would probably favor `args` or even removing these two output lines entirely if there aren't other `brew` commands that do the same thing.